[2025-06-10T22:43:59.471+0700] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-10T22:43:59.546+0700] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: earthquake_data_ingestion.check_api_connectivity scheduled__2025-06-08T23:00:00+00:00 [queued]>
[2025-06-10T22:43:59.579+0700] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: earthquake_data_ingestion.check_api_connectivity scheduled__2025-06-08T23:00:00+00:00 [queued]>
[2025-06-10T22:43:59.584+0700] {taskinstance.py:2856} INFO - Starting attempt 1 of 3
[2025-06-10T22:43:59.642+0700] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): check_api_connectivity> on 2025-06-08 23:00:00+00:00
[2025-06-10T22:43:59.656+0700] {standard_task_runner.py:72} INFO - Started process 15446 to run task
[2025-06-10T22:43:59.670+0700] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'earthquake_data_ingestion', 'check_api_connectivity', 'scheduled__2025-06-08T23:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/ingestion.py', '--cfg-path', '/tmp/tmpmk3ktdn0']
[2025-06-10T22:43:59.680+0700] {standard_task_runner.py:105} INFO - Job 2: Subtask check_api_connectivity
[2025-06-10T22:43:59.953+0700] {task_command.py:467} INFO - Running <TaskInstance: earthquake_data_ingestion.check_api_connectivity scheduled__2025-06-08T23:00:00+00:00 [running]> on host DESKTOP-1U3MHTH.
[2025-06-10T22:44:00.456+0700] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='magnitudr-team' AIRFLOW_CTX_DAG_ID='earthquake_data_ingestion' AIRFLOW_CTX_TASK_ID='check_api_connectivity' AIRFLOW_CTX_EXECUTION_DATE='2025-06-08T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-08T23:00:00+00:00'
[2025-06-10T22:44:00.463+0700] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-10T22:44:01.141+0700] {ingestion.py:59} INFO - üîç Checking USGS API connectivity...
[2025-06-10T22:44:02.261+0700] {ingestion.py:78} INFO - ‚úÖ API connectivity OK - Found 1 test records
[2025-06-10T22:44:02.267+0700] {python.py:240} INFO - Done. Returned value was: True
[2025-06-10T22:44:02.386+0700] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-10T22:44:02.390+0700] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=earthquake_data_ingestion, task_id=check_api_connectivity, run_id=scheduled__2025-06-08T23:00:00+00:00, execution_date=20250608T230000, start_date=20250610T154359, end_date=20250610T154402
[2025-06-10T22:44:02.464+0700] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-06-10T22:44:02.570+0700] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-10T22:44:02.580+0700] {local_task_job_runner.py:240} INFO - ::endgroup::
